documentation/modules/con-customizing-labels-and-annotations.adoc

```diff
@@ -28,4 +28,6 @@ template:
 The `labels` and `annotations` fields can contain any labels or annotations that do not contain the reserved string `strimzi.io`.
 Labels and annotations containing `strimzi.io` are used internally by {ProductName} and cannot be configured.
 
+For Kafka Connect, annotations on the `KafkaConnect` resource are used to enable the creation and management of connectors using `KafkaConnector` resources. For more information, see xref:proc-enabling-kafkaconnectors-deployment-configuration-kafka-connect[].
```

Comment:
I wonder if this should be mentioned in the KafkaConnector resources section?

Reply:
Good suggestion. I've edited the **KafkaConnector resources** section to refer to using annotations.